dev-python/gitana/gitana-0.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="SQL-based Git Repository Inspector"
+HOMEPAGE="https://github.com/SOM-Research/Gitana"
+SRC_URI="https://github.com/SOM-Research/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+KEYWORDS="~amd64 ~x86"
+
+# problem with https://github.com/SOM-Research/Gitana/issues/5
+# wait for release of git-python version 2.0.9 and update as soon as possible
+
+DEPEND="dev-lang/python:2.7[tk]
```

Comment:
@timeraider4u For instance see https://github.com/gentoo-science/sci/blob/f69e73182fc7cdc6c600a72fceb88ae1626db25d/sci-misc/nltk/nltk-2.0.4.ebuild#L7